WISE-Community/WISE
# Fixed bug where report was showing up in personal space

This fixes the bug where the report item was showing up in the personal space after inserting a note into the report. See #1151 for how to reproduce.

How to test:
1. Insert personal note into report. The report item should not show up in personal space.
2. Insert public note into report. The report item should not show up in personal space.
